charts/chirpstack/templates/postgres/statefulset.yaml

```diff
@@ -0,0 +1,58 @@
+{{- if not .Values.postgres.existingPsqlUrl -}}
```

Comment:
I don't think we should deploy/manage our own postgres, we should either use an operator for this or force the user to give us an existing database connection